glide.yaml

```diff
@@ -11,7 +11,7 @@ import:
 # Pinned to a branch with w/a for https://github.com/giantswarm/kvm-operator/pull/216.
 # Make sure to rebase k8scloudconfig "kvm-operator-disable-rbac" with "master"
 - package: github.com/giantswarm/k8scloudconfig
-  version: 459a19f48292e3481fa1aff8104105b3c4fd2698
+  version: kvm-operator-disable-rbac
```

Comment:
Why should we disable RBAC again? Doesn't make sense to me.

Reply:
Read the comment above, It was like this on master before my merge. I'm currently on that PR to make the encryption and go back to master for `k8scloudconfig`

Reply:
You should in general make sure you get the right refs from master. Your original PR was weeks old and you did not properly merge master apparently. This here is the wrong version.

Reply:
the previous version of the file before my commit is this one https://github.com/giantswarm/kvm-operator/commit/b18f2e2990efd69dfe14e4cf3384bad12cd246fb#diff-395f41b2a27b70ce44399c91c82158c2

it's not master branch

Reply:
Julien, please take the version from master branch. I do not mean you should use `master` here. I mean you should check which version of k8scloudconfig is in master of kvm-operator. You should always merge against the current master. Now you just set the version of master as it was when you your original PR initially. This was weeks ago.

Reply:
and the original PR was not the ecryption one (i'm currently working on) but the etcd_data-dir fix https://github.com/giantswarm/kvm-operator/pull/273 taht is rom yesterday. So I don't get your point

Reply:
We talked. I was confused. Go ahead if this is what it should be.